libssh2-sys/lib.rs

```diff
@@ -33,6 +33,10 @@ pub const LIBSSH2_FLAG_COMPRESS: c_int = 2;
 pub const LIBSSH2_HOSTKEY_TYPE_UNKNOWN: c_int = 0;
 pub const LIBSSH2_HOSTKEY_TYPE_RSA: c_int = 1;
 pub const LIBSSH2_HOSTKEY_TYPE_DSS: c_int = 2;
+pub const LIBSSH2_HOSTKEY_TYPE_ECDSA_256: c_int = 3;
```

Comment:
@alexcrichton here's a `libssh2-sys` change for your review; just some trivial constant exposure